### ISSUE TITLE
Silence INFO from tflite

### ISSUE THREAD

**Issue by rikardn** [pharmpy/pharmpy#1326]
I think we should silence the following message:

`INFO: Created TensorFlow Lite XNNPACK delegate for CPU.`

**Comment by pharmpy-dev-123** [comment]
There does not seem to be a simple upstream solution:

  - https://github.com/google/mediapipe/issues/2094#issuecomment-862840734
  - https://github.com/google/mediapipe/issues/2119#issuecomment-860927310

**Comment by rikardn** [comment]
No, I think the message is still there, but I haven't checked in a while.

**Comment by misterBart** [comment]
Can be silenced in C++ API with tflite::LoggerOptions::SetMinimumLogSeverity(tflite::TFLITE_LOG_SILENT); 
https://github.com/tensorflow/tensorflow/issues/58050
Not sure about other APIs.
